src/Certificate.php

```diff
@@ -70,9 +62,13 @@ private function load()
     public function isExpired()
     {
         $now = new \DateTime('now');
-        return $this->validFrom <= $now && $this->validTo >= $now;
+        return $this->validTo < $now;
```

Comment:
isso aqui não pode ser assim, porque o certificado tem um período de início e fim